lib/zold/commands/remote.rb

```diff
@@ -112,6 +114,10 @@ def run(args = [])
       when 'update'
         update(opts)
         update(opts, false)
+      when 'select'
+        update(opts)
+        update(opts, false)
```

Comment:
@botanicus no need to do `update` here. Select should work only with the local file, don't touch the network.